src/components/common/GreenhouseEffectsSettings.js

```diff
@@ -0,0 +1,49 @@
+import { Typography } from '@material-ui/core';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import SliderWithLabel from './SliderWithLabel';
+import { setGreenhouseGasesValues } from '../../actions';
+
+const GreenhouseEffectsSettings = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const values = useSelector(({ lab }) => lab.greenhouseGasesValues);
+  const { ch4, co2, h2o } = values;
+
+  const onChange = (value, key) => {
+    dispatch(setGreenhouseGasesValues({ [key]: value }));
+  };
+
+  return (
+    <>
+      <Typography variant="h6">{t('Greenhouse Effects')}</Typography>
+      <SliderWithLabel
+        text={t('CH_4 (ppm)', { escapeInterpolation: true })}
+        max={10}
+        value={ch4}
+        onChange={(e, v) => onChange(v, 'ch4')}
+      />
+      <SliderWithLabel
+        text={t('H_2O (ppm)', { escapeInterpolation: true })}
```

Comment:
did not know that _ makes the text subscript...!

Reply:
Not at all^^ The first argument is actually a key, so you can input anything you want. If you check the translations you will see I added the subscript tag directly!